mars/services/meta/api/oscar.py

```diff
@@ -138,20 +138,22 @@ async def set_chunk_meta(
         memory_size: int = None,
         store_size: int = None,
         bands: List[BandType] = None,
+        timestamp: int = None,
```

Comment:
The design purpose for `timestamp` field need to be illustrated in detail.

Reply:
It's a version which can be an optimistic lock for set/del meta for same object. If  version is not None and not same, delete chunk meta will be skipped